lib/internal/worker.js

```diff
@@ -506,6 +509,32 @@ class Worker extends EventEmitter {
       };
     });
   }
+
+  // TODO(theanarkh): add options
```

Comment:
Would be helpful to expand the comment a bit more to describe the options you have in mind.

Reply:
Done !